src/aircloak/Aircloak/JsonApi/ResponseTypes/AircloakType.cs

```diff
@@ -0,0 +1,67 @@
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Since we don't want to add XML comments to all public methods, it will be better to just disable this warning in the project file, so we can avoid adding this pragma to most of the files. (https://stackoverflow.com/questions/29552452/how-to-hide-c-sharp-warning-using-project-csproj-file)

Reply:
On a second thought, using the pragma actually allows more granular settings for this warning.

Reply:
I wasn't sure - adding XML comments to all public methods can be a chore (especially when they are obvious) but in general it's good practice. In this particular case, the classes in question are simply collections of properties so I decided to disable the warning. For now we can continue using pragmas. If we find we are using them too much... then we can decide to disable it project-wide.